about.html

```diff
@@ -12,58 +12,96 @@
     
-    How We Work
+    <h1>Strategy</h1>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <p>Iterative approaches to corporate strategy foster collaborative 
+    thinking to further the overall value proposition.</p>
 
-    Learn More
+    <button>Learn More</button>
+    <div class="lines"></div> 
+    </div>
 
+    <div class="box">
+    <img class="boximg" src="img/about-working.png" alt="strategy">
+    
+    <h1>How We Work</h1>
+    
+    <p>Iterative approaches to corporate strategy foster collaborative 
+    thinking to further the overall value proposition.</p>
 
+    <button>Learn More</button>
+    <div class="lines"></div> 
+    </div>
 
-    <img src="img/about-office.png" alt="strategy">
+    <div class="box">
+    <img class="boximg" src="img/about-office.png" alt="strategy">
     
-    Places We Work
+    <h1>Places We Work</h1>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <p>Iterative approaches to corporate strategy foster collaborative 
+    thinking to further the overall value proposition.</p>
 
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
+    <button>Learn More</button>
+    <div class="lines"></div> 
+    </div>
+    
     
-    Collaboration
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <div class="box">
+    <img class="boximg" src="img/about-meeting.png" alt="strategy">
+    
+    <h1>Collaboration</h1>
+    
+    <p>Iterative approaches to corporate strategy foster collaborative 
+    thinking to further the overall value proposition.</p>
 
-    Learn More
+    <button>Learn More</button>
+    <div class="lines"></div> 
+    </div>
+    
+    </section>
 
-    Let's Work Together
+    
+    <h3><strong>Let's Work Together</strong></h3>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <p class="a">Leverage agile frameworks to provide a robust synopsis for high level overviews. 
+      Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. 
+      Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
         
       <footer>
         <nav>
           <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
+          <a href="about.html">About</a>
+          <a href="products.html">Products</a>
           <a href="#">Blog</a>
           <a href="#">Contact</a>
         </nav>
```

Comment:
Try to indent your code across your HTML and CSS it makes your code cleaner and easier to read.